grunt/tasks/jest.js

```diff
@@ -6,25 +6,87 @@
 
 'use strict';
 
+var assign = require('object-assign');
+var async = require('async');
+var fs = require('fs');
+var glob = require('glob');
 var grunt = require('grunt');
 var path = require('path');
 
+var rootPath = path.resolve('.');
+var buildPath = path.join(rootPath, 'build');
+var tempConfigPath = path.join(buildPath, 'jest-config.json');
```

Comment:
Creates a temp `jest-config.json` in the `react/build` path so that it's ignored by git.